docs/_includes/ex-header-attr.adoc

```diff
@@ -20,7 +20,7 @@ My document provides...
 // tag::b-attr[]
 :homepage: http://asciidoctor.org
 :docslink: http://asciidoctor.org/docs[Asciidoctor's Docs]
-:description: Asciidoctor is a mature, plain-text document format for +
+:summary: Asciidoctor is a mature, plain-text document format for +
```

Comment:
The choice to use `description` as the attribute name is important because it gets mapped to the description meta element in the HTML output.